sonar-plugin-server/src/main/java/jetbrains/buildserver/sonarplugin/manager/BaseSQSInfo.java

```diff
@@ -75,14 +75,14 @@ public String getUrl() {
 
     @Override
     public boolean isTokenLoginUsed() {
-        final String useToken = get(USE_TOKEN);
-        return useToken != null ? Boolean.parseBoolean(useToken) : get(TOKEN) != null;
+        final String useToken = getToken();
+        return useToken != null ? Boolean.parseBoolean(useToken) : getToken() != null;
     }
 
     @Nullable
     @Override
     public String getToken() {
-        return get(TOKEN);
+        return get(TOKEN) != null ? get(TOKEN) : get("token");
```

Comment:
Some comments as to why we're reading some other parameter with other name, could be helpful here.